app.js

```diff
@@ -2,45 +2,68 @@ let readBtn = document.querySelector('button');
 let audio;
 let inputHTML = document.querySelector('input');
 let outputHTML = document.querySelector('p');
-let ones = ['', 'یەک', 'دوو', 'سێ', 'چوار', 'پێنج', 'شەش', 'حەوت', 'هەشت', 'نۆ'];
-let teens = ['دە', 'یانزە', 'دوانزە', 'سیانزە' ,'چواردە' ,'پانزە' ,'شانزە', 'حەڤە', 'هەژدە', 'نۆزدە'];
-let tens = ['','','بیست','سی','چل','پەنجا','شەست','حەفتا','هەشتا','نەوە'];
+let numbers = {
+    ones: ['', 'یەک', 'دوو', 'سێ', 'چوار', 'پێنج', 'شەش', 'حەوت', 'هەشت', 'نۆ'],
+    teens: ['دە', 'یانزە', 'دوانزە', 'سیانزە' ,'چواردە' ,'پانزە' ,'شانزە', 'حەڤە', 'هەژدە', 'نۆزدە'],
+    tens: ['','','بیست','سی','چل','پەنجا','شەست','حەفتا','هەشتا','نەوە']
+}
+
+inputHTML.addEventListener('input', (e) => {
+    outputHTML.innerHTML = "ئەنجام: " + convertToWords(e.target.value);
+    console.log(e);
+});
 
-inputHTML.addEventListener('input', () => {
-    outputHTML.innerHTML = "ئەنجام: " + convertToWords(inputHTML.value);
+readBtn.addEventListener('click', ()=> {
+    let number = inputHTML.value;
+    readNumber(number);
 });
 
 function playAudio(number) {
     audio = new Audio(`audio/${number}.m4a`);
     audio.play();
 }
-readBtn.addEventListener('click', ()=> {
-    let number = inputHTML.value;
-    readHundreds(number);
-    function readTens(number) {
-        if (number < 20) { 
-            playAudio(number);
-        } 
-        else if(number > 19) {
-            if(number % 10 != 0) {
-                playAudio(Math.floor(number / 10) * 10);
+function readNumber(number) {
+    if(number < 0) 
```

Comment:
Move this logic inside readBtn event listener and call readHundreds from there. Then rename readHundred to readNumber and remove this function. It's cleaner that way.